packages/core/web-test-runner.config.mjs

```diff
@@ -10,6 +10,13 @@ import { fromRollup } from '@web/dev-server-rollup';
 import execute from 'rollup-plugin-shell';
 import baseConfig from './web-dev-server.config.mjs';
 
+// import the browser launcher you want to use, chromeLauncher is the default
+import { chromeLauncher } from '@web/test-runner';
+// import { playwrightLauncher } from '@web/test-runner-playwright';
```

Comment:
Do we still need these commented lines?

Reply:
Nope, I'll revert back to whats in next. I was learning how the new test harness worked.